src/oncall/user_sync/ldap_sync.py

```diff
@@ -283,8 +285,13 @@ def sync(config, engine):
         contacts['full_name'] = row.full_name
         contacts['photo_url'] = row.photo_url
 
+    # all oncall users
     oncall_usernames = set(oncall_users)
 
+    # set of inactive oncall users
+    rows = engine.execute('SELECT name FROM user WHERE active = FALSE')
+    inactive_usernames = set((user.name for user in rows))
```

Comment:
Last comment: the double parens aren't needed if the comprehension is the sole argument for a function call. Eg just `set(user.name for user in rows)` would work.